versioned_docs/version-2.9/how-to-guides/advanced-user-guides/enable-user-retention.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Enabling User Retention
+---
+
+<head>
+  <link rel="canonical" href="https://ranchermanager.docs.rancher.com/how-to-guides/advanced-user-guides/enable-user-retention.md"/>
+</head>
+
+In Rancher v2.8.5 and later (and v2.7.14 and later in the v2.7.x line), you can enable user retention to automatically disable or delete inactive user accounts after a configurable time period.
```

Comment:
```suggestion
You can enable user retention to automatically disable or delete inactive user accounts after a configurable time period.
```

We don't need to call out patch versions because this will be available from the first release of 2.9 and onward.